src/features/leda-nft/services/item.service.ts

```diff
@@ -43,12 +43,12 @@ export default class ItemService extends HttpService {
 
   async findAllHistory(): Promise<History[]> {
     const { data } = await this.instance.get<History[]>(`${this.endpoint}/history`);
-    return data;
+    return data.reverse();
```

Comment:
How does the history endpoint sort the results? May I ask why you are using reverse here ? I'm just curious